cloudformation/media-atom-maker.json

```diff
@@ -236,6 +241,47 @@
                 ]
             }
         },
+        "UserUploadBucket": {
+            "Type": "AWS::S3::Bucket",
+            "Properties": {
+                "CorsConfiguration": {
+                    "CorsRules": [{
+                        "AllowedHeaders": ["*"],
+                        "AllowedMethods": ["GET", "PUT"],
+                        "AllowedOrigins": [
+                          { "Fn::Join": ["", [ "https://", { "Ref": "DomainToMonitor" } ]] }
+                        ]
+                    }]
+                }
+            }
+        },
+        "LimitedUploadRole": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "AssumeRolePolicyDocument": {
+                    "Statement": [{
+                        "Effect": "Allow",
+                        "Principal": {
+                            "AWS": { "Fn::Join": ["", [ "arn:aws:iam::", { "Ref": "AWS::AccountId" }, ":root"]] }
```

Comment:
No harm in putting this here, put you only need to do principal delegation when you're going across AWS accounts. Within a single account, normal IAM permissions are enough.